### PR TITLE
Add wrld.bg, World Bank's new shortener and 9 other domains

### PR DESCRIPTION
**Pull request by konieczkow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.4.0] - 2017-07-25
```
Comment (ascastro) [inline CHANGELOG.md]: This should be today's date :)

File: README.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
 
 A Ruby library to expand shortened URLs.
 
-**Current version:** 1.3.0  
+**Current version:** 1.4.0
```
Comment (mudge) [inline README.md]: Can you keep the trailing spaces here as they are required for Markdown line breaks?

File: CHANGELOG.md (hunk 1 above)
Comment (konieczkow) [reply CHANGELOG.md]: It was bound to happen

Comment (konieczkow) [reply CHANGELOG.md]: Ah, I knew I would forget something! Thanks!

**Pull request merged**